_chapters/chapter_07.md

```diff
@@ -19,7 +19,7 @@ layout: rails_tutorial
 
 {% steps %}
 {% list %}
-  1.  Download the stylesheet file `bookstore.scss` [here](link-to-file.com).
+  1.  Download the stylesheet file `bookstore.scss` [here]({{site.baseurl}}/assets/downloads/bookstore.scss).
```

Comment:
I *might* have one potential downside of hosting the stylesheet here. If we change the the stylesheet in the bookstore app, we have to remember to also update it here. We could end up with two different stylesheets.

I was kinda thinking we'd have a completed version of the bookstore app tied to the completed version of tutorial. If we did that, we could link to the stylesheet from the bookstore app.

I dunno if this is a big enough issue to worry about, but it was just a thought I had.